src/pb_convert.rs

```diff
@@ -156,39 +147,68 @@ impl ProtobufConvertStruct {
     }
 }
 
+impl ProtobufConvertFieldAttrs {
+    fn impl_field_setter(&self, ident: &Ident) -> impl ToTokens {
+        let pb_getter = Ident::new(&format!("get_{}", ident), Span::call_site());
+
+        let setter = match (self.skip, &self.with) {
```

Comment:
I think without double-matching this block will be more readable, something like

```rust
if self.skip {
    return default_variant;
};

match &self.with {
    Some(with) => ...,
    None => ...
}
```